packages/core-scroll/core-scroll.js

```diff
@@ -109,7 +109,7 @@ export default class CoreScroll extends HTMLElement {
 
 function onMousedown (event) {
   if (closest(event.target, NEEDS_MOUSEDOWN)) return
-  if (!isLeftMouseButtonClick(event)) return
+  if (event.button !== 0) return // Only react to left clicking
```

Comment:
I had this as a function with a meaningfull name so we would not need comments like this.

Reply:
skjønner tanken, men siden funksjonen var så liten inliner jeg den like gjerne. vi bruker gjerne kommentarer fritt i core der det trengs